examples/elf/tests/mksymtab.sh

```diff
@@ -22,7 +22,7 @@ fi
 # Extract all of the undefined symbols from the ELF files and create a
 # list of sorted, unique undefined variable names.
 
-varlist=`find ${dir} -executable -type f | xargs nm | fgrep ' U ' | sed -e "s/^[ ]*//g" | cut -d' ' -f2 | sort | uniq`
```

Comment:
Can we merge the similar script into one:
apps/tools/mksymtab.sh

Reply:
maybe we can. but i'm not going to do it today.
it seems each of them generates slightly different code to avoid conflicting symbols.

Reply:
Maybe we can modify the code to use the same name, I don't see any benefit that each project have the different symbol table name.

Reply:
I am merging this.  I presume that the suggested enhancements would be a different PR.